gremlin/ui/backend.py

```diff
@@ -113,6 +114,8 @@ def activate_gremlin(self, activate: bool):
         else:
             # Stop running the code
             self.runner.stop()
+            player = AudioPlayer()
```

Comment:
I usuallz do `Singleton().method()`, i.e. `AudioPlayer().stop()` for these things when I know that I don't need the thing again.

Reply:
n/p